snapshots/overlay/overlay_test.go

```diff
@@ -344,26 +339,29 @@ func testOverlayView(t *testing.T, newSnapshotter testsuite.SnapshotterFunc) {
 	if m.Source != "overlay" {
 		t.Errorf("mount source should be overlay but received %q", m.Source)
 	}
-
-	expectedOptions := 2
-	userxattr, err := NeedsUserXAttr(root)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if userxattr {
-		expectedOptions++
-	}
-
-	if len(m.Options) != expectedOptions {
-		t.Errorf("expected %d additional mount option but got %d", expectedOptions, len(m.Options))
-	}
-	lowers := getParents(ctx, o, root, "/tmp/view2")
-	expected = fmt.Sprintf("lowerdir=%s:%s", lowers[0], lowers[1])
-	optIdx := 1
-	if userxattr {
-		optIdx++
+	indexOffOnce.Do(getIndexOff)
+	if indexOff {
+		if len(m.Options) != 2 {
```

Comment:
instead of hard-coding `2`, possibly construct an `expected` slice for this test as well (similar to the other test), then you can check if `if len(m.Options) != len(expected) {` ?